CMakeLists.txt

```diff
@@ -26,8 +29,7 @@ set(EXTRA_LDFLAGS)
 #  etc. See https://github.com/libsdl-org/SDL/issues/4150
 add_library(sdl-build-options INTERFACE)
 
-if(WINDOWS_STORE)
-  cmake_minimum_required(VERSION 3.11.0)
+if(WINDOWS_STORE AND ( (CMAKE_VERSION VERSION_GREATER 3.11.0) OR  (CMAKE_VERSION VERSION_EQUAL 3.11.0) ) )
```

Comment:
This is tricky, because `-ZW` is really needed for UWP. [See this](https://learn.microsoft.com/en-us/cpp/build/reference/zw-windows-runtime-compilation?view=msvc-170).

Btw, why is the `cmake_minimum_required` here?
[`target_compile_options`](https://cmake.org/cmake/help/v3.0/command/target_compile_options.html) are [`target_compile_definitions`](https://cmake.org/cmake/help/v3.0/command/target_compile_definitions.html) are supported by CMake 3.0.

[The release notes of CMake 3.11 don't mention anything Windows relevant](https://cmake.org/cmake/help/latest/release/3.11.html).

Reply:
> Btw, why is the cmake_minimum_required here?

I'm not too sure. It appeared to originate from this commit:
https://github.com/microsoft/vcpkg/commit/8dc8d0e0c97550a95b764287adbad90b7df7d11d  

Maybe the version check should be removed.

Reply:
Let's remove the version check.
People targeting UWP with older CMake versions will probably have other issues as well.